packages/devtools_app/lib/src/screens/deep_link_validation/validation_details_view.dart

```diff
@@ -62,14 +62,14 @@ class ValidationDetailView extends StatelessWidget {
                     viewType == TableViewType.singleUrlView)
                   _PathCheckTable(controller: controller),
                 const SizedBox(height: extraLargeSpacing),
-                if (linkData.domainErrors.isNotEmpty)
-                  Align(
-                    alignment: Alignment.bottomRight,
-                    child: FilledButton(
-                      onPressed: () async => await controller.validateLinks(),
-                      child: const Text('Recheck all'),
-                    ),
+                Align(
+                  alignment: Alignment.bottomRight,
+                  child: FilledButton(
+                    onPressed: () async =>
+                        await controller.loadAndroidAppLinksAndValidate(),
```

Comment:
nit: I think this can this just be `onPressed: controller.loadAndroidAppLinksAndValidate,`